specs/generic.yaml

```diff
@@ -0,0 +1,35 @@
+openapi: "3.0.2"
+info:
+  title: version api
+  version: "1.0"
+paths:
+  /orchestrator/k8s/resource/info:
```

Comment:
this API is not generic. (you are sending hardcoded inception pod name in this API)

Reply:
file name is generic only, we can renamed but it contains apis which are used for generic purpose inception pod name is not hardcoded, its one of the generic api from this file. response is based on label selector.